packages/jest-snapshot/src/__tests__/InlineSnapshots.test.ts

```diff
@@ -73,6 +73,36 @@ expect(a).toMatchInlineSnapshot(\`[1, 2]\`);
   );
 });
 
+test('saveInlineSnapshots() with bad prettier path leaves formatting outside of snapshots alone', () => {
```

Comment:
This is literally a copy/paste of the previous test case, but with a phony install path.